lib/nimble.phx.gen.template/project.ex

```diff
@@ -23,11 +15,14 @@ defmodule Nimble.Phx.Gen.Template.Project do
             api_project?: false,
             web_project?: false,
             live_project?: false,
-            erlang_asdf_version: @default_versions[:erlang_asdf_version],
-            elixir_asdf_version: @default_versions[:elixir_asdf_version],
-            elixir_mix_version: @default_versions[:elixir_mix_version],
-            docker_build_base_image: @docker_base_images[:build],
-            docker_app_base_image: @docker_base_images[:app]
+            elixir_version: @elixir_version,
+            erlang_version: @erlang_version,
+            node_version: @node_version,
+            elixir_asdf_version:
+              "#{@elixir_version}-otp-#{@erlang_version |> String.split(".") |> List.first()}",
+            docker_build_base_image:
+              "hexpm/elixir:#{@elixir_version}-erlang-#{@erlang_version}-alpine-#{@alpine_version}",
+            docker_app_base_image: "alpine:#{@alpine_version}"
```

Comment:
Since there are many many atributes now 😆 do you think it would be better to order them alphabetically or have groupings:

```
defstruct otp_app: nil,
          # Dependency Versions 
          elixir_version: @elixir_version,
          erlang_version: @erlang_version,
          node_version: @node_version,
          elixir_asdf_version: "#{@elixir_version}-otp-#{@erlang_version |> String.split(".") |> List.first()}",
          # Variants
          api_project?: false,
          web_project?: false,
          live_project?: false,
```

Also having documentation for attributes like `otp_app` and `base_module` to better understand what this does would be good. This can be added in the moduledoc in another PR.

Reply:
yeah 👍 I have fixed it in https://github.com/nimblehq/elixir-templates/pull/79/commits/e05f49bae25ec883d6d325b2b8576b41e1f88678 :)